python/kfserving/requirements.txt

```diff
@@ -6,7 +6,7 @@ urllib3>=1.15.1
 kubernetes>=12.0.0
 tornado>=6.0.0
 argparse>=1.4.0
-minio>=4.0.9
+minio>=4.0.9,<7.0.0
```

Comment:
I think this effectively means `minio==6.0.2`, so make it more explicit?

Reply:
I think having `minio==6.0.2` will make some users have conflicts with their other packages. How about `minio>=4.0.9,<=6.0.2`?

Reply:
I think only storage initializer currently are depending on minio.

Reply:
yes, but kfserving SDK also using the same `requirements.txt` where the conflicts will kick in.

Reply:
I brought up this because pip is changing how they handle the package dependencies
https://pip.pypa.io/en/latest/user_guide/#deprecation-timeline

Reply:
ok the pip is going to be more strict.